x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.test.ts

```diff
@@ -596,7 +607,37 @@ describe('Response actions', () => {
         expect(responseBody.action).toBeUndefined();
       });
 
-      it('handles execute without optional `timeout`', async () => {
+      it('handles execute with `0` timeout', async () => {
```

Comment:
yeah, not sure what supporting `0` gives us

Reply:
Yeah, I was not sure about this, but I assumed that a user might want to give an input such as `0s`? Are we only allowing `> 0` values for timeout?

Reply:
Yeah, I'm not following or understanding why a user would define `0s` for timeout. My suggestion is change it to be a min of `1`

Reply:
I'll change it back to 1.

Reply:
997bde19c8da4e7798793f8fcd7041902412a6f1